CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 2.50.0-rc.0 / 2024-02-07
+
+* [CHANGE] Remote Write: Error `storage.ErrTooOldSample` is now generating HTTP error 400 instead of HTTP error 500. #13335
+* [FEATURE] Remote Write: Drop old inmemory samples. Activated using the config entry `sample_age_limit` #13002
+* [FEATURE] **Experimental**: Add support for ingesting zeros as created timestamps (enabled under the feature-flag `created-timestamp-zero-ingestion`). #12733 #13279
+* [FEATURE] Promtool: Add `analyze` histograms command #12331
+* [FEATURE] TSDB/compaction: Add a way to enable overlapping compaction #13282 #13393 #13398
+* [FEATURE] Add automatic memory limit handling. Activated using the feature flag `auto-gomemlimit` #13395
+* [ENHANCEMENT] Promtool: allow setting multiple matchers to `promtool tsdb dump`. #13296
```

Comment:
While the above is what the PR title says, I found it a little confusing, especially the use of "to" ("setting" vs "specifying" doesn't really matter)

```suggestion
* [ENHANCEMENT] Promtool: allow specifying multiple matchers in `promtool tsdb dump`. #13296
```